pkg/utils/common_test.go

```diff
@@ -87,3 +95,85 @@ func TestGenerateKey(t *testing.T) {
 		})
 	}
 }
+
+func setupMultiConcurrentContexts(t *testing.T) func() {
+	tkgConfigFile, err := os.CreateTemp("", "config")
```

Comment:
Nit: let's stop calling these files with a `tkg` prefix.  We can just use `configFile` and `configFileNG`

Reply:
Agreed. Updated.